frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { match } from 'react-router';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+import { useDispatch } from 'react-redux';
+import { ActionGroup, Button, Form } from '@patternfly/react-core';
+import {
+  NodeKind,
+  StorageClassResourceKind,
+  referenceForModel,
+  k8sCreate,
+  K8sResourceKind,
+} from '@console/internal/module/k8s';
+import {
+  withHandlePromise,
+  HandlePromiseProps,
+  history,
+  ButtonBar,
+} from '@console/internal/components/utils';
+import { setFlag } from '@console/internal/actions/features';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { getName } from '@console/shared';
+import { minSelectedNode, defaultRequestSize } from '../../../constants/ocs-install';
```

Comment:
Update the index file and combine it with line 24

Reply:
There are other variables defined too inside `ocs-install.ts`. All needs to be updated and moved. Can be taken as a part of refactoring PR.